sdk/core/Azure.Core/src/DiagnosticsOptions.cs

```diff
@@ -26,10 +30,21 @@ internal DiagnosticsOptions()
             LoggedQueryParameters = new List<string>();
         }
 
+        /// <summary>
+        /// Get or sets value indicating whether HTTP pipeline logging is enabled.
+        /// </summary>
         public bool IsLoggingEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets value indicating whether distributed tracing spans are going to be created for this clients methods calls and HTTP calls.
+        /// </summary>
         public bool IsDistributedTracingEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets value indicating whether the "User-Agent" header containing <see cref="ApplicationId"/>, client library package name and version, <see cref="RuntimeInformation.FrameworkDescription"/>
```

Comment:
nit:  I believe you missed "should be sent" or similar after "version" and before the first `<see>`